app/scripts/SFM_APP.xml.template

```diff
@@ -1,6 +1,6 @@
 
```

Comment:
@tanismar I would not add the SFM_APP.xml.template to the repo. This should be the same original SFM template with the flag for the bilateral filter on or off.